admin_manual/configuration_server/index_php_less_urls.rst

```diff
@@ -0,0 +1,49 @@
+Enable index.php-less URLs
+==========================
+
+Since ownCloud 9.0.3 you need to explicitly configure and enable index.php-less URLs
+(e.g. https://example.com/apps/files/ instead of https://example.com/index.php/apps/files/).
+The following documentation provides the needed steps to configure this for the ``Apache``
+Web server.
+
+Prerequisites
+-------------
+
+Before being able to use index.php-less URLs you need to enable the ``mod_rewrite`` and
+``mod_env`` Apache modules. Furthermore a configured ``AllowOverride All`` directive
+within the vhost of your Web server is needed. Please have a look at the manual of
+``Apache`` how to enable and configure these.
```

Comment:
"Please have a look at the Apache manual for how to enable and configure these."